### PR TITLE
Update to KtLint 0.34.2

### PR DESCRIPTION
**Pull request by arturbosch**

### CODE REVIEW

File: docs/pages/documentation/formatting.md [hunk 1]
```diff
@@ -82,10 +82,6 @@ See <a href="https://ktlint.github.io/#rule-blank">ktlint-website</a> for docume
 
 See <a href="https://ktlint.github.io/#rule-empty-class-body">ktlint-website</a> for documentation.
 
-### NoItParamInMultilineLambda
-
-See <a href="https://ktlint.github.io">ktlint-website</a> for documentation.
```
Comment (schalkms) [inline docs/pages/documentation/formatting.md]: Why did you remove this rule?

File: docs/pages/documentation/formatting.md [hunk 2]
```diff
@@ -164,10 +160,6 @@ See <a href="https://ktlint.github.io/#rule-spacing">ktlint-website</a> for docu
 
 See <a href="https://ktlint.github.io/#rule-spacing">ktlint-website</a> for documentation.
 
-### SpacingAroundUnaryOperators
-
-See <a href="https://ktlint.github.io/#rule-spacing">ktlint-website</a> for documentation.
```
Comment (schalkms) [inline docs/pages/documentation/formatting.md]: Same here

File: docs/pages/documentation/formatting.md (hunk 1 above)
Comment (arturbosch) [reply docs/pages/documentation/formatting.md]: Ktlint removed this due to no symbol solving I guess

File: docs/pages/documentation/formatting.md (hunk 2 above)
Comment (arturbosch) [reply docs/pages/documentation/formatting.md]: This was also removed. It interfered with chain wrapping as I remember correctly from own tests
